packages/hooks/src/useCurrentRegion.js

```diff
@@ -2,26 +2,37 @@ import { useState, useEffect } from 'react';
 import { avRegionsApi } from '@availity/api-axios';
 
 export default () => {
-  const [currentRegion, setCurrentRegion] = useState(undefined);
+  const [currentRegion, setCurrentRegion] = useState();
   const [loading, setLoading] = useState(true);
+  const [error, setError] = useState();
 
-  const fetchCurrentRegion = async () => {
-    const response = await avRegionsApi.getCurrentRegion();
+  useEffect(() => {
+    let ignore = false;
 
-    if (response.data.regions[0] !== undefined) {
-      setCurrentRegion({
-        code: response.data.regions[0].id,
-        value: response.data.regions[0].value,
-      });
-    }
+    const fetchCurrentRegion = async () => {
+      setLoading(true);
+      try {
+        const response = await avRegionsApi.getCurrentRegion();
 
-    setLoading(false);
-  };
+        if (!ignore) {
+          setCurrentRegion({
+            code: response.data.regions[0].id,
+            value: response.data.regions[0].value,
+          });
+        }
+      } catch (error_) {
+        if (!ignore) setError(error_);
+      }
+
+      setLoading(false);
```

Comment:
Just as a suggestion, we can add a `finally` block here that assures the `setLoading` happens after the `try/catch`. Otherwise LGTM